openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -461,6 +649,18 @@ class CoursewareInformation(RetrieveAPIView):
         * linkedin_add_to_profile_url: URL to add the effective user's certificate to a LinkedIn Profile.
         * user_needs_integrity_signature: Whether the user needs to sign the integrity agreement for the course
         * learning_assistant_enabled: Whether the Xpert Learning Assistant is enabled for the requesting user
+        * content_type_gating_enabled: Whether the content type gating is enabled for the course
+        * show_calculator: Whether the calculator should be shown in the course details page
+        * can_access_proctored_exams: Whether the user is eligible to access proctored exams
+        * notes: An object containing note settings for the course
+            * enabled: Boolean indicating whether edxnotes feature is enabled for the course
+            * visible: Boolean indicating whether notes are visible in the course
+        * marketing_url: The marketing URL for the course
+        * studio_url: The URL to the course in Studio, if the user is staff
+        * is_cosmetic_price_enabled: Boolean indicating whether the cosmetic price feature is enabled
+        * overview: The overview HTML content for the course
+        * ocw_links: A list of OpenCourseWare links for the course
+        * prerequisites: A list of prerequisite courses for the course
```

Comment:
I managed to get data for the "prerequisites" field through the settings on the Schedule & Details, but on the legacy page nothing is displayed in the sidebar. This may indicate that the API is returning data in an incorrect format. Could you check this?

[Template](https://github.com/openedx/edx-platform/blob/master/lms/templates/courseware/course_about.html#L247)

```
    "prerequisites": [
        "course-v1:openedx2+123+2024"
    ]
```